### PR TITLE
Created test file for ScoreResource and tests run successfully

### LINKED ISSUE #11: Create automated test cases for score form
In test cases:
- Create a user
- Create a document
- Submit scores for a document under created user
- Check that the scores are saved

### PR DESCRIPTION
**Pull request by claralam**
Wrote TestScoreResource.java and ran the tests successfully.
The tests create a user, document, and has the user add a new score to check if the POST and GET API calls work.

Resolves #11

### CODE REVIEW

File: docs-web/src/test/java/com/sismics/docs/rest/TestScoreResource.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import com.sismics.util.filter.TokenBasedSecurityFilter;
+
+
+/**
+ * Tests of the score resource.
+ * 
+ */
+public class TestScoreResource extends BaseJerseyTest {
+    /**
+     * Test the comment resource.
```
Comment (joannetsaii) [inline docs-web/src/test/java/com/sismics/docs/rest/TestScoreResource.java:23]: Should be changed to `score resource` or remove this comment.

**Review by joannetsaii — APPROVED** [review]
Minor comment issue. Overall LGTM!

File: docs-web/src/test/java/com/sismics/docs/rest/TestScoreResource.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        Assert.assertEquals(1, json.getJsonArray("scores").size());
+        JsonObject score = json.getJsonArray("scores").getJsonObject(0);
+        Assert.assertEquals(score1Id, score.getString("id"));
+        Assert.assertEquals(3, score.getInt("skills"));
+        Assert.assertEquals(5, score.getInt("experience"));
+        Assert.assertEquals(2, score.getInt("transcriptGPA"));
+        Assert.assertEquals(4, score.getInt("match"));
+        Assert.assertNotNull(score.getJsonNumber("create_date"));
+    }
+}
```
Comment (ChaosZhai) [inline docs-web/src/test/java/com/sismics/docs/rest/TestScoreResource.java:76]: LGTM!

**Review by ChaosZhai — APPROVED** [review]

**Comment by hiVictoria** [comment]
looks good!

**Review by hiVictoria — COMMENTED** [review]
looks good! (accidentally commented when I meant to review before merging pull request)

**Pull request merged**